src/bindings/calib/stereocalibration_bindings.cpp

```diff
@@ -18,6 +18,14 @@ namespace py = pybind11;
 void init_stereocalibration(py::module_ &m) {
     NDArrayConverter::init_numpy();
     // Load in stereo camera calibration file
+    py::enum_<I3DR::Phase::CalibrationSelection>(m, "CalibrationSelection", R"(
+        Structure of CalibrationSelection
+
+        )")
+        .value("LEFT", I3DR::Phase::CalibrationSelection::LEFT)
+        .value("RIGHT", I3DR::Phase::CalibrationSelection::RIGHT)
+        .export_values();
+    
     py::class_<I3DR::Phase::StereoCameraCalibration>(m, "StereoCameraCalibration", R"(
             Variables of stereo camera calibration file
```

Comment:
I think "Variables of stereo camera calibration file" should be changed to "Store and manipulate stereo camera calibration data"

Reply:
Done